core/cautils/helmchart.go

```diff
@@ -76,10 +91,14 @@ func (hc *HelmChart) GetWorkloads(values map[string]interface{}) (map[string][]w
 		if len(wls) == 0 {
 			continue
 		}
-		// separate base path and file name. We do not use the os.Separator because the paths returned from the helm engine are not OS specific (e.g. mychart/templates/myfile.yaml)
```

Comment:
Is there a reason to remove the comment?

Reply:
I think I remove it with my own comment accidentally. Fix